### PR TITLE
created both html and css file for the homepage

### PR DESCRIPTION
**Pull request by AjShatalii**

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Git Workflow Workshop</title>
```
Comment (mahmoudhammad309) [inline Index.html]: change the title of the, please

**Review by mahmoudhammad309 — CHANGES_REQUESTED** [review]

**Review by mahmoudhammad309 — CHANGES_REQUESTED** [review]
change the title of the web page, please

**Comment by AjShatalii** [comment]
all required changes have been done

**Review by mahmoudhammad309 — APPROVED** [review]
well done

**Pull request merged**